src/modules/iasql_functions/iasql.ts

```diff
@@ -322,6 +323,22 @@ ${Object.keys(tableCollisions)
         dbId,
       );
     }
+    (async () => {
+      const user = await MetadataRepo.getUserFromDbId(dbId);
+      if (user && (moduleNames.length > 0 || moduleNames[0] !== 'aws_account')) {
```

Comment:
what about the rest of list? if it's a `iasql_install('aws_account', 'aws_s3')`. Are we relying on the fact that after installing account, we'll need to sync the regions?

Reply:
oh good point

Reply:
this is to ignore the dashboard setup of a database where it calls `aws_account` by itself